doc/releases/changelog-dev.md

```diff
@@ -119,6 +119,9 @@
 
 * The `qml.clifford_t_decomposition` has been improved to use less gates when decomposing `qml.PhaseShift`.
   [(#6842)](https://github.com/PennyLaneAI/pennylane/pull/6842)
+ 
+* `qml.qchem.taper` now handles wire ordering for the tapered observables more robustly.
+  [(#6954)](https://github.com/PennyLaneAI/pennylane/pull/6954)
```

Comment:
Maybe should be in bug fix section instead?

Reply:
Won't call it a bug myself, but can shift it there.